code/modules/client/preferences_toggles.dm

```diff
@@ -340,3 +340,13 @@ GLOBAL_LIST_INIT(ghost_others_options, list(GHOST_OTHERS_SIMPLE, GHOST_OTHERS_DE
 
 	show_popup_menus = !show_popup_menus
 	to_chat(src, "<span class='interface'>Right click [show_popup_menus ? "en" : "dis"]abled.</span>")
+
+ //Same thing as the character creator preference, but as a byond verb, because not everyone could reach it
+/client/verb/toggle_tgui_fancy()
```

Comment:
```suggestion
///Same thing as the character creator preference, but as a byond verb, because not everyone can reach it in tgui preference menu
/client/verb/toggle_tgui_fancy()
```